lib/report_plugins/glucosedistribution.js

```diff
@@ -122,7 +122,18 @@ glucosedistribution.report = function report_glucosedistribution (datastorage, s
   var data = datastorage.allstatsrecords;
   var days = datastorage.alldays;
 
-  $('#glucosedistribution-days').text(days + ' ' + translate('days total'));
+  // Peter Leimbach (begin)
```

Comment:
Can you remove these lines that add your name and other comments unrelated to the logic, would prefer to not merge them. LGTM otherwise